tests/app/conftest.py

```diff
@@ -1219,5 +1221,16 @@ def delete(endpoint, _expected_status=204, **endpoint_kwargs):
     return AdminRequest
 
 
+@pytest.fixture(scope='function')
+def app_statsd():
+    current_app.config['NOTIFY_ENVIRONMENT'] = "test"
+    current_app.config['NOTIFY_APP_NAME'] = "api"
+    current_app.config['STATSD_HOST'] = "localhost"
+    current_app.config['STATSD_PORT'] = "8000"
+    current_app.config['STATSD_PREFIX'] = "prefix"
+    current_app.statsd_client = Mock()
+    return current_app
```

Comment:
Wondering if you could reuse the fixture from utils to avoid duplicating this. I think that in other tests StatsD is mocked with `mocker` directly if you want to avoid adding a new fixture

Reply:
Thanks, I am not super familiar with these Python test libraries so any hint is appreciated. I'll try to apply the changes.

Reply:
Regarding reusing the fixtures from *utils*: it seems unfortunately that the module shadowing occurring by both `conftest` files in the same `tests` folder will not allow this.